agent/util-scripts/gold/test-client-tool-meister/test-56.txt

```diff
@@ -341,11 +322,70 @@ INFO pbench-tool-meister wait -- waiting for stop mpstat
 INFO pbench-tool-meister send_tools -- testhost.example.com: send_tools (no-op) lite /var/tmp/pbench-test-utils/pbench/mock-run/1-iter-one/sample42/tools-lite/testhost.example.com
 INFO pbench-tool-meister sysinfo -- pbench-sysinfo-dump -- /var/tmp/pbench-test-utils/opt/pbench-agent/unittest-scripts/pbench-sysinfo-dump /var/tmp/pbench-test-utils/pbench/mock-run/sysinfo/end block,security_mitigations,sos 
 INFO pbench-tool-meister sysinfo -- testhost.example.com: sysinfo send (no-op) lite /var/tmp/pbench-test-utils/pbench/mock-run/sysinfo/end
-INFO pbench-tool-meister main -- terminating
-INFO pbench-tool-meister main -- Remove pid file ... (tm-lite-testhost.example.com.pid)
---- mock-run/tm/tm-lite-testhost.example.com.log file contents
+INFO pbench-tool-meister driver -- terminating
+--- mock-run/tm/tm-lite-testhost.example.com.err file contents
 +++ mock-run/tm/tm-lite-testhost.example.com.out file contents
 --- mock-run/tm/tm-lite-testhost.example.com.out file contents
++++ mock-run/tm/tm.logs file contents
```

Comment:
@RobertKrawitz, this is how the combined Tool Meister logs will look in the `${benchmark_run_dir}/tm/tm.logs` file.

The registered host name is emitted with a sequence number so that the log file can be searched / sorted as appropriate.

Reply:
:+1: The sequence number is very helpful.